.github/workflows/deploy-happy-stack.yml

```
@@ -41,10 +41,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Update deployment
-        env:
-          TFE_TOKEN: ${{ secrets.TFE_TOKEN }}
-        run: |
-          ./scripts/happy --profile="" --env ${{ github.event.deployment.environment }} update ge${{ github.event.deployment.environment }}stack --tag ${{ github.event.deployment.payload.tag }}
+        uses: chanzuckerberg/github-actions/.github/actions/deploy-happy-stack@v1.3.0
+        with:
+          tfe-token: ${{ secrets.TFE_TOKEN }}
+          stack-name: ge${{ github.event.deployment.environment }}stack
+          happy_version: "0.20.0"
+          create-tag: "true"
```

```suggestion
          create-tag: "false"
```
I think the images should already be created at this point